aws/scripts/py/s3_sandbox_configuration.py

```
@@ -0,0 +1,65 @@
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "*",
+                },
+                "Action": "s3:GetObject",
+                "Resource": [
+                    f"arn:aws:s3:::{PROJECT_NAME}-{BRANCH_NAME}/*",
+                    f"arn:aws:s3:::{PROJECT_NAME}-{BRANCH_NAME}",
+                ],
+            },
+        ]
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)